unbxdAutosuggest.js

```diff
@@ -1104,6 +1112,11 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 					+ "&rows=" + this.options.popularProducts.count + "&"
 					+ default_search_params;
 
+			if (self.options.popularProductsFields.length > 0) {
+				var popular_product_fields = "&fields=" + self.options.popularProductsFields.join(",");
```

Comment:
As per w3c in javascript, variable should be named using camelCase convention: https://www.w3schools.com/JS/js_conventions.asp. Please stick to the same standards

Reply:
all of the other url variables were named in this format, ex: default_search_params, url_path, etc. Hence kept the same format..